vision/Object Detection/object_bounding_box.py

```diff
@@ -0,0 +1,74 @@
+"""
+Find the bounding box of the contour and crop down the image to include only the object of interest.
+"""
+import cv2
+import numpy as np
+import argparse
+import random
+
+
+def get_bounding_image(img, contour):
```

Comment:
Can you type annotate this function: img, contour, what it returns 

```
def get_bounding_image(img: np.array, contour: np.array) -> np.array:
```

Reply:
Everything else looks good otherwise

Reply:
Suggested changes have been made.